salesagility/SuiteCRM
# Fixed #7928 - Upgrade wizard recommends composer update instead of composer install

## Description



Updated a language string recommending that users run ```composer update``` instead of the standard ```composer install --no-dev``` for production environments.

## Motivation and Context

Issue reference: #7928 

## How To Test This

1. Run through an upgrade via the upgrade wizard.
2. See that it recommends running composer install.

## Types of changes


### Final checklist
